### PR TITLE
[Documentation] Update Virtual Assistant section

### PR DESCRIPTION
**Pull request by Batta32**
Related to 3689

### Purpose
*What is the context of this pull request? Why is it being done?*
The [virtual-assistant](https://github.com/microsoft/botframework-solutions/tree/master/docs/_docs/virtual-assistant) section of the public documentation is outdated

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
- Migration from botbuilder-tools to botframework-cli commands
- Remove redundant information
- Fix broken/empty links

The updated documents are the followings:
- docs/_docs/virtual-assistant/handbook/activity-flow.md
- docs/_docs/virtual-assistant/handbook/deployment-scripts.md
- docs/_docs/virtual-assistant/handbook/devops.md
- docs/_docs/virtual-assistant/handbook/localization.md
- docs/_docs/virtual-assistant/tutorials/create-assistant/csharp/3-create-project.md
- docs/_docs/virtual-assistant/tutorials/create-assistant/csharp/4-provision-your-azure-resources.md
- docs/_docs/virtual-assistant/tutorials/create-assistant/typescript/3-create-project.md
- docs/_docs/virtual-assistant/tutorials/create-assistant/typescript/4-provision-you-azure-resources.md
- docs/_docs/virtual-assistant/tutorials/customize-assistant/csharp/2-edit-your-greeting.md
- docs/_docs/virtual-assistant/tutorials/customize-assistant/csharp/3-edit-responses.md
- docs/_docs/virtual-assistant/tutorials/customize-assistant/csharp/4-edit-your-cognitive-models.md
- docs/_docs/virtual-assistant/tutorials/customize-assistant/typescript/2-edit-your-greeting.md
- docs/_docs/virtual-assistant/tutorials/customize-assistant/typescript/3-edit-responses.md
- docs/_docs/virtual-assistant/tutorials/customize-assistant/typescript/4-edit-your-cognitive-models.md
- docs/_docs/virtual-assistant/tutorials/deploy-assistant/cli/1-intro.md
- docs/_docs/virtual-assistant/tutorials/deploy-assistant/cli/4-update-application-settings.md
- docs/_docs/virtual-assistant/tutorials/deploy-assistant/cli/5-deploy-luis-models.md
- docs/_docs/virtual-assistant/tutorials/deploy-assistant/cli/7-create-dispatch-model.md
- docs/_docs/virtual-assistant/tutorials/deploy-assistant/web/1-intro.md
- docs/_docs/virtual-assistant/tutorials/deploy-assistant/web/4-update-application-settings.md
- docs/_docs/virtual-assistant/tutorials/deploy-assistant/web/7-create-dispatch-model.md

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
It was rendered manually in order to check the result of the changes

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [X] I have updated related documentation

### CODE REVIEW

**Review by VictorGrycuk — APPROVED** [review]

File: docs/_docs/virtual-assistant/handbook/devops.md [hunk 1]
```diff
@@ -30,47 +30,36 @@ When you want to test changes to your LUIS models and QnA Maker knowledgebases i
...
 
-1. Check in your updated .lu files to source control. 
+1. Check in your updated `.lu` files to source control. 
     > Your changes should go through a peer review to validate there will be no conflicts. You can also share your LUIS app and/or transcripts of the bot conversation with your changes to help in this conversation.
 
 1. Run your build pipeline to deploy your updated files to your production environment. 
     > This pipeline should update your LUIS models, QnA Maker knowledgebases, and Dispatch model as needed.
 
-## I've changed my skill LUIS model. What next?
+## I've changed my skill LUIS model. Whats next?
```
Comment (matiasroldan6) [inline docs/_docs/virtual-assistant/handbook/devops.md:41]: `What's`

File: docs/_docs/virtual-assistant/tutorials/customize-assistant/csharp/2-edit-your-greeting.md [hunk 2]
```diff
@@ -13,78 +13,78 @@ order: 2
...
-```
+        ],
+        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+        "version": "1.0",
+        "speak": "${NewUserIntroCardTitle()}"
+    }
+    ```
 
-2. In your assistant's project, navigate to **Responses** > **MainResponses.lg** and find the `# NewUserIntroCardJson` section. Paste the above JSON inbetween the ``` separators.
+1. In your assistant's project, navigate to **Responses** > **MainResponses.lg** and find the `# NewUserIntroCardJson` section. Paste the above JSON inbetween the ``` separators.
```
Comment (fran893) [inline docs/_docs/virtual-assistant/tutorials/customize-assistant/csharp/2-edit-your-greeting.md:85]: inbetween -> in between

**Review by fran893 — CHANGES_REQUESTED** [review]

**Review by fran893 — APPROVED** [review]

File: docs/_docs/virtual-assistant/tutorials/create-assistant/typescript/3-create-project.md [hunk 3]
```diff
@@ -11,107 +11,97 @@ order: 3
...
 
 ### Generate the assistant using prompts
-
-- The generator will start prompting for some information that is needed for generating the sample:
   - `What's the name of your assistant? (sample-assistant)`
-        > The name of your assistant (also used as your project's name and for the root folder's name).
-    - `What's the description of your assistant? ()`
-        > The description of your assistant.
-    - `Which languages will your assistant use? (by default takes all the languages)`
+      > The name of your assistant (used also as your project's name and for the root folder's name)
```
Comment (matiasroldan6) [inline docs/_docs/virtual-assistant/tutorials/create-assistant/typescript/3-create-project.md:25]: I think `also used` was ok. But it's super minimal, don't change it back

**Review by matiasroldan6 — APPROVED** [review]

File: docs/_docs/virtual-assistant/tutorials/create-assistant/typescript/3-create-project.md (hunk 3 above)
Comment (Batta32) [reply docs/_docs/virtual-assistant/tutorials/create-assistant/typescript/3-create-project.md:25]: I can do it!

**Pull request merged**